proto-public/pbconnectca/ca.pb.binary.go

```diff
@@ -1,5 +1,5 @@
 // Code generated by protoc-gen-go-binary. DO NOT EDIT.
-// source: proto-public/pbconnectca/ca.proto
+// source: pbconnectca/ca.proto
```

Comment:
Super trivial, but did we want it to know the full path?

Reply:
I _think_ its entirely unnecessary. For stack traces and things you will have the full path to the generated file just like you do today.

The reason for this is due to how `buf` handles modules. The module root here is proto-public and `buf` uses paths relative to the module root.